### PR TITLE
CSI 4732 vg svc implementation

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

**Review by ELENAGER — APPROVED** [review]
LGTM

File: controllers/tests/array_action/svc/array_mediator_svc_test.py [hunk 1]
```diff
@@ -2067,3 +2072,84 @@ def _test_delete_host_rmhost_errors(self, client_error, expected_error):
...
+        return Munch(volume_group)
+
+    def _prepare_lsvolumegroup(self, no_return=None, volume_count=0):
+        if no_return:
+            return_value = None
+        else:
+            return_value = self._mock_cli_volume_group(volume_count=volume_count)
+        self.svc.client.svcinfo.lsvolumegroup.return_value = Mock(as_single_element=return_value)
+
+    def test_create_volume_group(self):
```
Comment (matancarmeli7) [inline controllers/tests/array_action/svc/array_mediator_svc_test.py:2092]: can you add `success` and `failed` prefixes to the test functions?

Comment (ArbelNathan) [reply controllers/tests/array_action/svc/array_mediator_svc_test.py]: done

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Pull request merged**